NewTutorials/Tutorial/PlayerViewControllerWrapper.swift

```diff
@@ -77,6 +79,17 @@ class PlayerViewControllerWrapper: UIViewController {
                     return props
                 }
                 
+                if props.shouldObserve {
+                    _ = player.addObserver(on: DispatchQueue.global(qos: .background)) { (props) in
+                        guard let item = props.playbackItem else { return }
+                        let isStreamPlaying = item.ad.isStreamPlaying || item.content.isStreamPlaying
+                        let currentTime = item.ad.time.static?.current ?? item.content.time.static?.current ?? 0
+                        
+                        print("isPlaying:", isStreamPlaying)
```

Comment:
Can you render this changes on the UI as we discussed yesterday?

Reply:
Of course. Planned to add it today